src/sagas/wallet.js

```diff
@@ -97,12 +100,22 @@ export function* isPushNotificationEnabled() {
  * @returns {Generator<unknown, boolean>}
  */
 export function* isWalletServiceEnabled() {
+  // Users might have had issues with the wallet-service in the past, we can detect
```

Comment:
If the `IGNORE_WS_TOGGLE_FLAG` changed, how can we detect old flags with this code?

Reply:
You are right, I shouldn't change the flag, fixed in 2e37c2cccc323d9b6239d835e723d80fa26dd040